docs/docsite/rst/installation_guide/installation_distros.rst

```diff
@@ -143,10 +143,32 @@ For more details, see `this AskUbuntu post <https://askubuntu.com/a/1307181>`_.
 Also note that, for security reasons, we do NOT add the key to ``/etc/apt/trusted.gpg.d/``
 nor to ``/etc/apt/trusted.gpg`` where it would be allowed to sign releases from ANY repository.
 
+Installing Ansible on Arch Linux
+--------------------------------
+
+To install the full ``ansible`` package run:
```

Comment:
```suggestion
To install the ``ansible`` package run:
```

Reply:
This is the same wording as the Fedora example. I'd prefer to keep the clarifying adjectives.

Reply:
Ah, I scrolled right past your rationale for these suggestions :). I'd vote to keep the adjectives here and change everything en masse in a separate PR if we decide to do so.